Formula/thrax.rb

```diff
@@ -32,10 +31,8 @@ class Thrax < Formula
   fails_with gcc: "5"
 
   # Fix -flat_namespace being used on Big Sur and later.
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-big_sur.diff"
-    sha256 "35acd6aebc19843f1a2b3a63e880baceb0f5278ab1ace661e57a502d9d78c93c"
-  end
+  # patch ref, https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-big_sur.diff
+  patch :DATA
```

Comment:
When none of our patches for the `-flat_namespace` bug apply, we usually just regenerate `configure` with `autoreconf --force --install --verbose`.

Reply:
cool, I will give a try.

Reply:
@carlocab Should be good to go.